src/node/markdown/plugins/highlightLines.ts

```diff
@@ -9,16 +10,34 @@ export const highlightLinePlugin = (md: MarkdownIt) => {
     const [tokens, idx, options] = args
     const token = tokens[idx]
 
-    // due to use of markdown-it-attrs, the {0} syntax would have been converted
-    // to attrs on the token
+    // due to use of markdown-it-attrs, the {0} syntax would have been
+    // converted to attrs on the token
     const attr = token.attrs && token.attrs[0]
+
+    let lines = null
+
     if (!attr) {
-      return fence(...args)
+      // markdown-it-attrs maybe disabled
+      const rawInfo = token.info
+
+      if (!rawInfo || !RE.test(rawInfo)) {
+        return fence(...args)
+      }
+
+      const langName = rawInfo.replace(RE, '').trim()
+
+      // ensure the next plugin get the correct lang
+      token.info = langName
```

Comment:
@brc-dd Should this plugin be used after `attrs`? It changes `token.info`, `attrs` seems not working to me.

<img width="177" alt="image" src="https://user-images.githubusercontent.com/8336744/174726207-1c068f94-f597-4d5e-9101-4709a08a60c3.png">

<img width="299" alt="image" src="https://user-images.githubusercontent.com/8336744/174726233-83967c42-db47-4c12-8dbd-0c83a817c448.png">

Reply:
This wasn't working on v0 too IG. Can you create a separate issue for this? Probably the attributes on `code` are getting removed somewhere.

Reply:
@brc-dd Done #826